File: 2212-removing-minimum-and-maximum-from-array/removing-minimum-and-maximum-from-array.py
class Solution:
    def minimumDeletions(self, nums: List[int]) -> int:
        n = len(nums)
        minInd = nums.index(min(nums))
        maxInd = nums.index(max(nums))
        i, j = min(minInd, maxInd), max(minInd, maxInd)
        return min(j + 1, n - i, (i + 1) + (n - j))

File: 2212-removing-minimum-and-maximum-from-array/removing-minimum-and-maximum-from-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var minimumDeletions = function(nums) {
    const n = nums.length;
    let minInd = 0, maxInd = 0;
    let mini = nums[0], maxi = nums[0];
    for (let i = 1; i < n; i++) {
        if (nums[i] < mini) { mini = nums[i]; minInd = i; }
        if (nums[i] > maxi) { maxi = nums[i]; maxInd = i; }
    }
    const i = Math.min(minInd, maxInd);
    const j = Math.max(minInd, maxInd);
    return Math.min(j + 1, n - i, (i + 1) + (n - j));
};